mlflow/server/auth/__init__.py

```diff
@@ -8,15 +8,59 @@
 """
 
 import logging
+import uuid
+import os
+from pathlib import Path
+from flask import Flask, request, make_response, Response, redirect, flash, render_template_string
 
 from mlflow.server import app
+from mlflow.server.auth.config import read_auth_config
+from mlflow.server.auth.sqlalchemy_store import SqlAlchemyStore
+from mlflow.server.handlers import _get_rest_path
 
+_AUTH_CONFIG_PATH_ENV_VAR = "MLFLOW_AUTH_CONFIG_PATH"
 
 _logger = logging.getLogger(__name__)
 
+store = SqlAlchemyStore()
+
+
+class ROUTES:
+    USERS = _get_rest_path("/mlflow/users")
+    SIGNUP = _get_rest_path("/mlflow/signup")
```

Comment:
Can we change the route of the sign-up page to `/signup` instead of `api/2.0/mlflow/signup`?

Reply:
sure! just the signup page?

Reply:
Yep, just the signup page.